front/itiltemplatefield.form.php

```diff
@@ -46,6 +46,11 @@
 $item_class = $itiltype . 'Template' . $fieldtype . 'Field';
 $item = new $item_class;
 
+$item_field = $item_class::$items_field;
```

Comment:
I would have put this `$items_field` in `CommonITILObject` and its subclasses and name it `$itemlinkclass` (as we already have `$userlinkclass`, `$grouplinkclass` and `$supplierlinkclass`).

So you could asse it using :
```suggestion
$itil_type = $item_class::$itiltype;
$item_field = $itil_type::$itemlinkclass;
```

By the way, your code will fail if `$fieldtype` is different from `Predefined`.


EDIT: You could also use `getForeignKeyFieldForItemType($itil_type::getItemClass())`

Reply:
Done in c962154.